x/auth/vesting/keeper/grpc_query.go

```diff
@@ -0,0 +1,49 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	"github.com/cosmos/cosmos-sdk/types/query"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
+)
+
+func (vk VestingKeeper) VestingAccounts(c context.Context, req *types.QueryVestingAccountsRequest) (*types.QueryVestingAccountsResponse, error) {
```

Comment:
We should define this on a `Querier` which just embeds the `VestingKeeper`. This is the pattern we've been trying to establish in modules 👍